src/panels/lovelace/editor/hui-glance-card-editor.ts

```diff
@@ -0,0 +1,115 @@
+import { html, LitElement } from "@polymer/lit-element";
+import "@polymer/paper-checkbox/paper-checkbox.js";
+
+import processConfigEntities from "../common/process-config-entities";
+import { hassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { HomeAssistant } from "../../../types.js";
+import { LovelaceConfig, LovelaceCardEditor } from "../types.js";
+import { fireEvent } from "../../../common/dom/fire_event.js";
+
+import "../../../components/entity/state-badge.js";
+import "../../../components/ha-card.js";
+import "../../../components/ha-icon.js";
+
+interface EntityConfig {
```

Comment:
why are we duplicating this ?

Reply:
okay, you're not going to believe this, are you ready? Pay very close attentino, ok? Here it goes: Add the word `export` in front if it in the Glance card.